### ISSUE TITLE
Cannot finetune en_ppstructure_mobile_v2.0_SLANet_train with paddlepaddle-GPU

### ISSUE THREAD

**Issue by tangh146** [PaddlePaddle/PaddleOCR#12519]
#### 问题描述 / Problem Description

_Finetuning en_ppstructure_mobile_v2.0_SLANet_train with paddlepaddle-GPU fails with error.
Finetuning with the CPU package works just fine._

#### 运行环境 / Runtime Environment
_- OS: Windows 10, Windows 11
- Paddle: 2.6.0, 2.6.1
- PaddleOCR: 2.7.5
- CUDA/CUDNN: 11.7/8.9.7, 12.0/8.9.7_

#### 复现代码 / Reproduction Code

_Dataset and annotation structure: PubTabNet
Loss function: SLALoss_

#### 完整报错 / Complete Error Message

_(InvalidArgument) Currently, the type of tensor in slice indices only allows int32 and int64, please check the type of index tensor. (at ..\paddle/fluid/pybind/slice_utils.h:70)_

#### 可能解决方案 / Possible solutions

_Finetuning with the CPU package works just fine._
